multinet/api/tests/test_upload_csv.py

```diff
@@ -58,15 +57,16 @@ def airports_csv(owned_workspace: Workspace, authenticated_api_client, s3ff_clie
 
 
 @pytest.mark.django_db
-def test_create_upload_model_csv(owned_workspace: Workspace, user: User, airports_csv):
+def test_create_upload_model_csv(workspace: Workspace, user: User, airports_csv):
     """Test just the response of the model creation, not the task itself."""
+    workspace.set_owner(user)
```

Comment:
Similar to my other comment
```suggestion
    workspace.set_user_permission(user, WorkspacePermission.writer)
```